red-hat-gallery/README.md

```diff
@@ -428,3 +498,40 @@ $ oc set env deploy/productcatalogservice EXTRA_LATENCY-
 Red Hat Gallery comes with Prometheus alerts which will be fired if the service latency is too high:
 
 ![Alerts](docs/images/gallery-alerts.png "Alerts")
+
+
+## Uninstalling Red Hat Gallery
+
+To uninstall you need to remove the artifacts which you installed earlier:
+
+```
+oc delete cm cluster-monitoring-config -n openshift-monitoring
```

Comment:
Can we start the command-line with "$" character to stay consistent with the rest of the document?

Reply:
I personally hate the leading '$' or '#' because it makes it more difficult to cut and paste